src/styles/card.scss

```diff
@@ -0,0 +1,128 @@
+.products {
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), grid-template-columns should not repeat the same size when only one column is needed. You can use '200px' without repeat function.